test/common/testvm.py

```diff
@@ -259,7 +273,8 @@ def _kill_ssh_master(self):
             self.ssh_master = None
         if self.ssh_process:
             self.ssh_process.stdin.close()
-            self.ssh_process.wait()
+            if not self._abort:
+                self.ssh_process.wait()
```

Comment:
> On test failure, we still use the ssh master connection to download logs from the machine.

That's also the case on non-timeout failures. My question is why the timeout is different from other failures with regards to downloading the logs.